man/systemd.mount.xml

```diff
@@ -161,8 +161,8 @@
 
     <para>The NFS mount option <option>bg</option> for NFS background mounts
     as documented in <citerefentry project='man-pages'><refentrytitle>nfs</refentrytitle><manvolnum>5</manvolnum></citerefentry>
-    is not supported in <filename>/etc/fstab</filename> entries. The systemd mount option <option>nofail</option>
-    provides similar functionality and should be used instead.</para>
+    is detected by systemd-fstab-generator and the options are transformed so
+    that systemd fulfills the job-control implications of that option.</para>
```

Comment:
I'd still point people to using nofail/automounts/whatever here, as "bg" still doesn't strike me as a great choice.